Resources/views/standard_layout.html.twig

```diff
@@ -142,7 +142,13 @@ file that was distributed with this source code.
                                                         {% if not loop.last  %}
                                                             <li>
                                                                 {% if menu.uri is not empty %}
-                                                                    <a href="{{ menu.uri }}">{{ menu.label|raw }}</a>
+                                                                    <a href="{{ menu.uri }}">
+                                                                        {% if menu.extra('safe_label', false) %}
```

Comment:
> I would name it safe_html and make it default to true for BC.

Uuuuh you're right, it should default to true. @ellementA , can you change `menu.extra('safe_label', false)` to `menu.extra('safe_label', true)`? By default, labels come from xliff files and should be considered safe. We want it to be false only when coming from the database.